BlockServer/test_modules/runcontrol_manager_tests.py

```diff
@@ -48,19 +48,24 @@ def add_block(cs, data):
     cs.add_block(data)
 
 
+def dummy_sleep_func(seconds):
+    return
+
+
+def _get_time(t, **kwargs):
+    td = timedelta(**kwargs)
+    return datetime.strftime(t + td, '%m/%d/%Y %H:%M:%S')
```

Comment:
`td` is created and only used once. I think it would be neater to use `timedelta` in place. The meaning is clear so readability isn't significantly reduced